Created Installer and Supporting Docs

noodlemctwoodle opened the pull request:
Created easy installer without the need to run Powershell and reduce
peoples security by disabling script execution policy. Installer.exe
is a simple light weight installer the puts all the correct files in
the correct location. 

The following changes have been added to the Repo.

- Added Installer.exe
- Updated Readme.md with instructions for installing with Installer.exe
- Added Wiki entry for compiling installer yourself (trust but verify)
- Added compiler script for open source compiler Inno Setup 5.6.1

Resubmitting Pull Request with squashed changes and cleaner commits.

Commit:
Created Installed and Supporting Docs

Created easy installer without the need to run Powershell and reduce
peoples security by disabling script execution policy. Installer.exe
if a simple light weight installer the puts all the correct files in
the correct location. The following changes have been added to the Repo.

- Added Installer.exe
- Updated Readme.md with instructions for installing with Installer.exe
- Added Wiki entry for compiling installer yourself (trust but verify)
- Added compiler script for open source compiler Inno Setup 5.6.1

mikefitz888 commented:
Install.exe :: Scan Results
[VirusTotal (0 / 69)](https://www.virustotal.com/#/url/4986e0ba54e1b5404469bb80ea6e3a891e36b7216d5bfdb22a2e53cc003e4ebc/detection) 👍

mikefitz888 commented:
I cannot see the wiki entry yet

noodlemctwoodle commented:
You may need to carry it across from my [fork](https://github.com/noodlemctwoodle/Azure-Manic-7Pace/wiki/Compile-Inno-Installer), I can give you the code.

mikefitz888 commented on inno_compile-script:
```diff
@@ -0,0 +1,33 @@
+AppPublisherURL={#MyAppURL}
+AppSupportURL={#MyAppURL}
+AppUpdatesURL={#MyAppURL}
+DefaultDirName={localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin
```
This should be `...\Plugins\Packages\MFitz...`

noodlemctwoodle replied:
Updated to the correct path added in updated PR

mikefitz888 commented on readme.md:
```diff
@@ -15,19 +15,13 @@ It achieves this in 2 ways:
-Close ManicTime (The application can be hidden in the [Notification Area](https://docs.microsoft.com/en-us/windows/desktop/uxguide/winenv-notification), so ensure it has been completely exited)
-Copy the folder named `MFitzpatrick.TagSource.Azure7PacePlugin` to the plugin folder opened in the previous step.
+Download Install.exe and run as administrator. This will extact the plugin to the correct plugins directory for Manic Time.
+ `{localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin`
```
This should be updated inline with the correct install location.

noodlemctwoodle replied:
Now has the correct path in the latest PR.

Commit:
Fixed Installer Bug

Changed installer path to correct location.
- Added new installer folder path
- Added new installer icon
- Added updated Inno script with correct extension
- Updated readme.md with the correct path and removed typos
- Added Installer/readme.md for compiling the source code yourself
- Corrected file path mistake in Installer/readme.md

mikefitz888 commented on Installer/Inno_Setup_Compiler_Script.iss:
```diff
@@ -15,12 +12,12 @@ AppPublisher={#MyAppPublisher}
 DefaultGroupName={#MyAppName}
 AllowNoIcons=yes
-OutputDir=C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master
+OutputDir=C:\Users\Toby.Goulden\Downloads\Azure-Manic-7Pace-master
```
You can't just assume everyone has the same name 😛

noodlemctwoodle replied:
Removed username from the file path.

mikefitz888 commented on Installer/Inno_Setup_Compiler_Script.iss:
```diff
@@ -15,12 +12,12 @@ AppPublisher={#MyAppPublisher}
+OutputDir=C:\Users\Toby.Goulden\Downloads\Azure-Manic-7Pace-master
 OutputBaseFilename=Install
-SetupIconFile=C:\Windows\ACU.ico
+SetupIconFile=C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\icon.ico
```
Shouldn't this be a relative path?

mikefitz888 replied:
Nevermind, I think i mis-understood how it works.

mikefitz888 commented on Installer/Inno_Setup_Compiler_Script.iss:
```diff
@@ -30,4 +27,4 @@ Name: "english"; MessagesFile: "compiler:Default.isl"
 Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\PluginIcon.png"; DestDir: "{app}"; Flags: ignoreversion
 Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\PluginSpec.json"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\Lib\*"; DestDir: "{localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin\lib"; Flags: ignoreversion
+Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\Lib\*"; DestDir: "{localappdata}\Finkit\ManicTime\Plugins\Packages\\MFitzpatrick.TagSource.Azure7PacePlugin\lib"; Flags: ignoreversion
```
Double backslash `\\`

noodlemctwoodle replied:
Removed double backslash, updated PR

mikefitz888 commented:
Install.exe :: Scan Results (For update 2)
[VirusTotal (0/69)](https://www.virustotal.com/#/url/b27ea20b52b4d53e9267e7cba08ea4f136687c4aff05d5da9643e8eeb07a0a94/detection)
